.github/workflows/ci.yml

```diff
@@ -165,14 +165,14 @@ jobs:
           path: ${{ github.workspace }}/dist
           key: nginx-gateway-fabric-${{ github.run_id }}-${{ github.run_number }}
 
-  build:
-    name: Build Image
+  build-oss:
+    name: Build OSS images
     needs: [vars, binary]
     strategy:
       fail-fast: false
       matrix:
-        image: [ngf, nginx, plus]
-        platforms: ["linux/arm64, linux/amd64"]
+        image: [ngf, nginx]
+        platforms: ["linux/arm64, linux/amd64, linux/s390x, linux/ppc64le"]
```

Comment:
We can't build the N+ image with these architectures?

Reply:
NGINX Plus doesn't have packages for those architectures unfortunately

Reply:
Well to be more accurate, they build RHEL 9 and Ubuntu for `s390x`, but not for all the other distros.